ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js

```diff
@@ -159,7 +159,7 @@
     },
     {
       id: 'Facility Chaosphere',
-      regex: / 03:\y{ObjectId}:Added new combatant Chaosphere/,
+      regex: /03:\y{ObjectId}:Added new combatant Chaosphere/,
```

Comment:
I appreciate you trying to make lines consistent, but please try to keep pull requests focused on one thing, and not changing formatting elsewhere.  :)

Reply:
seams that i got a little bit confused Q_Q tried to revert all other changes i made and might changed to much. sry for the inconvinience